tasks/makeSearchIndex.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs')
+const utils = require('./utils/utils')
+const {parseProjects} = require('./utils/parseProjects')
+
+if (require.main !== module) {
+  module.exports = function(config) {
+    const targets = ['build/.temp/search_index.json']
+    const boards = parseProjects(config, false)
+    const boardsInfoPaths = boards.map(
+      b => `build/.temp/boards/${b.id}/info.json`
+    )
+
+    return {deps: boardsInfoPaths, targets, moduleDep: false}
+  }
+} else {
+  const {deps, targets} = utils.processArgs(process.argv)
+
+  const paths = deps
+  const indices = []
+
+  paths.forEach(p => {
+    const info = JSON.parse(fs.readFileSync(p).toString())
+    const index = {
+      id: info.id,
+      summary: info.summary,
+      bom: info.bom.lines
+        .map(l => l.description)
```

Comment:
I think we are missing out because you are only using the description. I tried our test case of searching "TL072ACP" and it comes up with nothing because the full name is only in the MPN field.

Reply:
Yes, MPN definitely. "TL072ACP"  was in the original issue description and hasn't been addressed (you should be testing that while developing). "atmega32u4" is another one where I can see projects not showing up that I know have that chip in them (Sangaboard v0.3). 

Maybe we could also including distributor SKUs, but I am less sure about that. I don't think I've ever seen anyone search for them in our analytics.